spring-cloud-gcp-data-firestore/src/test/java/org/springframework/cloud/gcp/data/firestore/it/FirestoreIntegrationTests.java

```diff
@@ -79,23 +83,30 @@ public void transactionTest() {
 		User alice = new User("Alice", 29);
 		User bob = new User("Bob", 60);
 
-		DefaultTransactionDefinition transactionDefinition = new DefaultTransactionDefinition();
-		transactionDefinition.setReadOnly(false);
-		TransactionalOperator operator = TransactionalOperator.create(this.txManager, transactionDefinition);
 
-		this.firestoreTemplate.save(alice)
-				.then(this.firestoreTemplate.save(bob))
-				.as(operator::transactional).block();
+		this.firestoreTemplate.save(alice).then(this.firestoreTemplate.save(bob)).block();
 
 		assertThat(this.firestoreTemplate.findAll(User.class).collectList().block())
 				.containsExactlyInAnyOrder(bob, alice);
 
+		DefaultTransactionDefinition transactionDefinition = new DefaultTransactionDefinition();
+		transactionDefinition.setReadOnly(false);
+		TransactionalOperator operator = TransactionalOperator.create(this.txManager, transactionDefinition);
+
+		reset(this.txManager);
 
 		// test rollback
```

Comment:
Should this method be called `rollbackTest` instead of `transactionTest`?

Reply:
I'm fixing it to test commit as well.